data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+					goto declining
+			action
+				set "unfettered: hesitant question"
+			`	"That is what we will be forced to do if you refuse, but we would prefer this to be kept a secret. So do you agree to do this for us?"`
+				goto choose
+			label trouble
+			action
+				"unfettered: hesitant question" ++
+			`	"That should not be a problem, the Hai will not bothered about you scanning their ship, if they even notice it," she says with a quick sniffing.`
+				goto choose
+			label design
+			action
+				set "unfettered: design question"
+			`	"We would like to know how they avoided vulnerabilities in the hull of a ship of such size, and the way cargo can be stored without making the ship defenseless. Extra space on a ship cannot only be used for cargo... I will tell you more about it later, once you bring us the scans."`
+				goto choose
+			label scan
+			action
+				set "unfettered: scan question"
+			`	"You need to use at least one of our Value Detectors. The information provided by it should be sufficient, as it goes beyond a simple cargo scan. If you don't have one, you can install one here. You may want to get multiple scanners so you can scan the ship from a longer range, and faster. Please do actively scan it. Passive intel gathered on the fly will not do."`
+				goto choose
+			label agree
+			`	On the way back to your ship, Mera runs up to you. "One... last... thing!" You let her catch her breath. "We have noticed one Geocoris just entered <waypoints> as expected, but seems to have some special modifications. Please scan that one specifically. We have marked its signature for you to recognize it. We want to have a detailed scan of its cargo. Also, I've just found a captain willing to give you two of our scanners. They will be placed in your cargo hold if there is no space on your ship, but do go to the lab's outfitter to make sure they are installed properly.`
+			action
+				outfit "Value Detector" 2
+				outfit "Cargo Scanner" 5
+				clear "unfettered: scan question"
+				clear "unfettered: design question"
+				clear "unfettered: hesitant question"
+			`	"I cannot stress enough that you NEED to use at least one Value Detector, as any other scanners will simply either not contain enough information, or we could be unable to make good use of it. Since the Geocoris is quite complex, it may take some time to complete the scan, so feel free to buy more scanners here. I'd recommend going to the lab to check if they are well connected to your ship."`
+			`	"Do note, however, that they can be supplemented by Cargo Scanners, which is why I have decided to provide a few of those to you as well. You can go to the outfitter to set it all up correctly."`
+				accept
+			label declining
+			action
+				"unfettered rep: Mera" --
+			`	She clench her fist, visibly angry for a moment, before calming down. "Alright, but it will take a while for me to get someone else to scan a Geocoris, and we're missing a significant opportunity."`
```

Comment:
```suggestion
			`	She clenches her fist, visibly angry for a moment, before calming down. "Alright, but it will take a while for me to get someone else to scan a Geocoris, and we're missing a significant opportunity."`
```